src/app/fyle/add-edit-expense/add-edit-expense.page.ts

```
@@ -3914,7 +3914,7 @@
     await fyCriticalPolicyViolationPopOver.present();
 
     const { data } = (await fyCriticalPolicyViolationPopOver.onWillDismiss()) as {
-      data: boolean;
+      data: boolean
```

ESLint

[error] Expected a semicolon.

(@typescript-eslint/member-delimiter-style)